editors/code/package.json

```diff
@@ -237,6 +237,14 @@
                     "default": false,
                     "markdownDescription": "Run `cargo check` on startup to get the correct value for package OUT_DIRs"
                 },
+                "rust-analyzer.cargo.rustc.defaultTarget": {
```

Comment:
```suggestion
                "rust-analyzer.rustc.defaultTarget": {
```

or

```suggestion
                "rust-analyzer.defaultTarget": {
```

Reply:
This is because I put the option into the `CargoConfig`. Maybe `rust-analyzer.cargo.defaultTarget` ?